### ISSUE TITLE
[T1] 요구사항 정의서 제출의 건

### ISSUE THREAD

**Issue by papercrane55123** [beyond-sw-camp/beyond-sw-camp-be01_last-dance#82]
[요구사항 정의서](https://drive.google.com/drive/folders/1ZvORxScSuYeJWJSI68ZbRbn1OLIFsIaa)

T1팀 요구사항 정의서 제출합니다.

저희 SSG팀의 경우에는 애자일 방법론을 채택하여 기획과 코드 작업, 테스트, 배포 과정을 빠르게 5~6회 정도 반복하겠다는 계획을 가지고 실제 작업중에 있습니다.

이와 같이 애자일 방법론을 채택해 작업하다보니 멘토님께서도 프로젝트 요구사항의 디테일, 분석이 조금 부족하다는 평을 들었습니다. 
저희 팀은 이를 보완하기 위해 SNS 기능에 알림 기능을 추가하거나 위키피디아, 나무위키와 같이 게시물 권한에 따라 조직 또는 이용자가 수정 및 작성할 수 있도록 계획하는 등 요구 사항 중 분류를 자세히 나눠 프로젝트의 정체성을 높이고자 노력했습니다.

금일은 저희 SSG팀 멘토링이 있는 날입니다. 한화 시스템에 가서 제출한 요구사항에 대해 피드백 받을 예정이며, 변경사항이 생기면 언제든 수정하여 그에 맞게 프로젝트 진행하도록 하겠습니다.

수정 사항을 코멘트에 남겨 주시면 참고하여 요구사항에 반영하도록 하겠습니다. 감사합니다.

**Comment by pySatellite** [comment]
애자일 - 기민하게 ... 의 가장 큰 장점은 "평" 을 빠르게 받을 수 있다는 것입니다. 그걸 또 유연하게 수용가능 한것이 핵심이지요 ^^
맨토님의 "프로젝트 요구사항의 디테일, 분석이 조금 부족하다는 평"에 대하여 기민하게 잘 운영 하고 있는 것으로 보입니다.

**Comment by papercrane55123** [comment]
네 맞습니다! 완성도를 낮추고 빠르게 평을 얻다보니 '고치는' 느낌보다는 '추가되는' 부분이 많다고 느껴졌습니다. 이러한 점에서 시간을 단축할 수 있었습니다. 디테일 등은 멘토링, Issue활용, PM 회의 등을 통해 지속적으로 보완하도록 하겠습니다.

요구사항 버전관리의 경우 T3와 같이 변경사항이 생기면 v(version)을 달리하여 공유문서함에 넣어 놓도록 하겠습니다.
감사합니다.
